feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/domain/locks/reusable/LocksChange.kt

```diff
@@ -22,3 +24,15 @@ fun MutableList<ReusableLock>.addIfPositive(type: ReusableLock.Type, amount: Bal
         add(ReusableLock(type, amount))
     }
 }
+
+fun List<LocksChange>.maximize(): LocksChange {
```

Comment:
This method wont wok in general, e.g. when the change is negative (e.g. when we unlock the tokens)
Given that I suggest to make it private in tinger gov logic to mark that it only intented to be used in that context
Or we can rename it to indicate that as well